src/tests/ftest/util/support_test_base.py

```diff
@@ -0,0 +1,134 @@
+"""
+(C) Copyright 2023 Intel Corporation.
+
+SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import os
+from datetime import datetime
+
+from control_test_base import ControlTestBase
+from run_utils import run_remote
+
+
+class SupportTestBase(ControlTestBase):
+    """Class for Support log collection """
+
+    def __init__(self, *args, **kwargs):
+        """Initialize a SupportTestBase object."""
+        super().__init__(*args, **kwargs)
+        self.target_folder = None
+        self.custom_log_dir = None
+        self.custom_log_file = None
+        self.custom_log_data = None
+        self.log_hosts = None
+        self.extract_dir = "/tmp/Extracted_support_log"
+
+    def create_custom_log(self, folder_name):
+        """Create custom log directory with custom data file on each servers.
+
+        Args:
+            folder_name (str): Name of the custom folder
+        """
+        server_custom_log = os.environ['AVOCADO_TESTS_COMMON_TMPDIR']
+        self.custom_log_dir = os.path.join(server_custom_log, folder_name)
+        self.custom_log_file = os.path.join(self.custom_log_dir, "Custom_File")
+        self.target_folder = os.path.join("/tmp", "DAOS_Support_logs")
```

Comment:
We should also use either `self.base_test_dir` or `self.test_dir` here for proper test cleanup and portability.